admin/acceptance.py

```diff
@@ -335,11 +339,16 @@ def configured_cluster_for_nodes(
     :returns: A ``Deferred`` which fires with ``Cluster`` when it is
         configured.
     """
+    default_volume_size = GiB(1)
+    if dataset_backend_configuration.get('auth_plugin') == 'rackspace':
+        default_volume_size = GiB(100)
```

Comment:
This looks like it should come from `node.agents.test.blockdevicefactory.MINIMUM_ALLOCATABLE_SIZES`.  Or maybe they should both come from a non-test file.